packages/core/src/App/Containers/AdditionalKycInfoModal/additional-kyc-info-modal.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { observer, useStore } from '@deriv/stores';
+import {
+    Modal,
+    DesktopWrapper,
+    MobileWrapper,
+    UILoader,
+    Button,
+    PageOverlay,
+    Div100vhContainer,
+} from '@deriv/components';
+import { Localize } from '@deriv/translations';
+
+export const AdditionalKycInfoModal = observer(() => {
+    const {
+        ui: { is_additional_kyc_info_modal_open: is_open, toggleAdditionalKycInfoModal },
+    } = useStore();
+
+    return (
+        <React.Suspense fallback={<UILoader />}>
+            <div className='additional-kyc-info-modal__container'>
+                <DesktopWrapper>
+                    <Modal
+                        has_close_icon
+                        is_open={is_open}
+                        title={
+                            <Localize i18n_default_text='Additional information required for Deriv MT5 account(s)' />
+                        }
+                        toggleModal={toggleAdditionalKycInfoModal}
+                        className='additinal-kyc-info'
+                        width='90rem'
+                    >
+                        <Modal.Body className='additional-kyc-info-modal__form'>
+                            <div>Form</div>
+                        </Modal.Body>
+                        <Modal.Footer has_separator>
+                            <Button primary onClick={toggleAdditionalKycInfoModal}>
+                                <Localize i18n_default_text='Submit' />
+                            </Button>
+                        </Modal.Footer>
+                    </Modal>
+                </DesktopWrapper>
+                <MobileWrapper>
+                    <PageOverlay
+                        is_open={is_open}
+                        portal_id='deriv_app'
+                        header={
+                            <Localize i18n_default_text='Additional information required for Deriv MT5 account(s)' />
```

Comment:
**Question**: Can a `<Text` tag be used with header? Then u can eliminatethe style `font-size: 1.4rem;`?